### PR TITLE
Fix fileinfo copy

### PR DESCRIPTION
**Pull request by jjanvier**
Contribution of @userz58 with the tests fixed and the CI launched.

| Q | A |
| --- | --- |
| Added Specs | Y |
| Added Behats | Y |
| Changelog updated | Y |
| Review and 2 GTM | Y here https://github.com/akeneo/pim-community-dev/pull/4083 |
| Micro Demo to the PO (Story only) |  |
| Migration script |  |
|  Tech Doc |  |

### CODE REVIEW

File: features/update/set_media.feature [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      | code       | type              | localizable | scopable | allowed_extensions | group |
+      | front_view | pim_catalog_image | 1           | 0        | jpg, png           | other |
+      | side_view  | pim_catalog_image | 0           | 1        | jpg, png           | other |
+      | user_guide | pim_catalog_image | 1           | 1        | png                | other |
+    And the following products:
+      | sku     |
+      | AKN_MUG |
+    Then I should get the following products after apply the following updater to it:
+      | product | actions                                                                                                            | result                                                                                                      |
+      | AKN_MUG | [{"type": "set_data", "field": "front_view", "data": "%fixtures%/SNKRS-1R.png", "locale": "fr_FR", "scope": null}] | {"values": {"front_view": [{"locale": "fr_FR", "scope": null, "data": "6/4/a/0/64a0e9d7c1a28112bbcbcbc06aa52a7032417744_SNKRS_1R.png"}]}} |
```
Comment (SamirBoulil) [inline features/update/set_media.feature]: I can't seem to have this step work ! Once the image product value is set onto the product.

It's data becomes the fileKey (generated at save time) so I don't quite see how to test the action / result. 

Maybe that's why this test is missing in the first place.

@skeleton Any idea :trollface: xD

**Comment by SamirBoulil** [comment]
Hello @userz58 !

I fixed some behats of this PR even though there is the set_media.feature I can't seem to make work.

I'll try to find a way around it. In the meantime if you have any suggestions feel free to contribute :)

**Comment by SamirBoulil** [comment]
I need to close this PR because it has been made on master and we will most likely merge it on 1.7.

File: features/update/set_media.feature (hunk 1 above)
Comment (skeleton) [inline features/update/set_media.feature]: for integration tests, we use https://github.com/akeneo/pim-community-dev/blob/master/tests/MediaSanitizer.php
to check if the result matchs with the pattern. Not the ideal, but we can't do much more unfortunately.
You can implement it for your behat.

Comment (SamirBoulil) [inline features/update/set_media.feature]: Thanks I'll try this ^^

**Comment by SamirBoulil** [comment]
There is a BC in this PR, so it should be rebased and merged in master.

**Comment by pierallard** [comment]
I rebased in on master, now you can go to ! https://github.com/akeneo/pim-community-dev/pull/6200
I close this PR, and open an new one.

**Pull request closed without merging**